src/darker/__main__.py

```diff
@@ -50,54 +46,44 @@ def format_edited_parts(
     """
     git_root = get_common_root(srcs)
     changed_files = git_diff_name_only(srcs, git_root)
-    head_srcs = {
-        src: git_get_unmodified_content(src, git_root) for src in changed_files
-    }
-    worktree_srcs = {src: (git_root / src).read_text() for src in changed_files}
-
-    # 1. run isort
-    if isort:
-        config = black_args.get("config")
-        line_length = black_args.get("line_length")
-        edited_srcs = {
-            src: apply_isort(edited_content, src, config, line_length)
-            for src, edited_content in worktree_srcs.items()
-        }
-    else:
-        edited_srcs = worktree_srcs
-
-    for src_relative, edited_content in edited_srcs.items():
-        max_context_lines = len(edited_content)
+    edited_linenums_differ = EditedLinenumsDiffer(git_root)
```

Comment:
I'd inline this or move this line to the only usage of this object (before line 70)